website_and_docs/content/blog/2022/end-of-year-review.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Year End Review 2022"
+linkTitle: "Year End Review 2022"
+date: 2022-12-20
+tags: ["selenium"]
+categories: ["general"]
+author: Corina Pip ([@imalittletester](https://twitter.com/imalittletester))
+description: >
+  Looking back at our achievements from 2022
+---
+
+It was an amazing year here at the Selenium project, and we wanted to remind you of some of the great things we accomplished.
+
+In terms of releases, we continued to build on top of Selenium 4 which was launched in October 2021. This year we
+published [4.2.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.2.0),
+[4.3.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.3.0), [4.4.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.4.0),
+[4.5.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.5.0), [4.6.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.6.0)
+and [4.7.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.7.0).
+These releases included improvements, bug fixes, removal of deprecated functionality, improved error handling,
+updates of the underlying libraries, support for the latest DevTools/BiDi
+versions to keep you in sync with new browser releases, but also new features.
+
+We introduced the first version (which we call beta) of the embedded Selenium Manager. The purpose of this feature is to
+help you manage your driver binaries without having to manually update them yourself each time a new version is released.
+We also introduced observability in the Docker-Selenium images. And we completed the implementation of the virtual
```

Comment:
I'd skip discussing virtual authenticator; no one really cares about it. It's mostly interesting because a different w3c standard is referencing webdriver spec, and it was a pain to figure out getting it to work in all the languages. 😄